charts/workload/templates/deployment.yaml

```diff
@@ -2,16 +2,22 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "workload.name" . }}
+  annotations:
+    {{- toYaml .Values.workload.annotations | nindent 4 }}
```

Comment:
We really should add these to the values.yaml file as sample defaults. 

```
workload:
  annotations: {}
  labels: {}
  containers:
    annotations: {}
    labels: {}
service:
  labels: {}
```

Why no annotations on the service?

Reply:
RE: `values.yaml` file as sample defaults - love the idea, but it's not the case today: https://github.com/score-spec/score-helm-charts/blob/main/charts/workload/values.yaml - useless for now, but let's add in another/dedicated PR?

Reply:
> Why no annotations on the service?
No reasons, I missed it, just added, thanks for catching this.